src/Makefile

```diff
@@ -425,11 +418,18 @@ MAC_WARNINGS_TO_FIX := 114,108
 MAC_WARNINGS_TO_FIX := $(MAC_WARNINGS_TO_FIX),4014
 
 MAC_COMMON_DEFINES = -define:NET_2_0,XAMARIN_MAC,MONOMAC,XAMCORE_2_0,__UNIFIED__
-MAC_full_ARGS = -sdk:4.5 -define:NO_SYSTEM_DRAWING -define:XAMMAC_SYSTEM_MONO
-MAC_mobile_ARGS = -nostdlib -r:mscorlib -lib:$(abspath $(MAC_DESTDIR)/Library/Frameworks/Xamarin.Mac.framework/Versions/Current/lib/mono/Xamarin.Mac)
+MAC_full_ARGS = -define:NO_SYSTEM_DRAWING -define:XAMMAC_SYSTEM_MONO
+MAC_mobile_ARGS = 
```

Comment:
No args needed for mobile?

Reply:
Also, where are these (both full and mobile args) actually used?

Reply:
Mobile is now covered by MAC_mobile_CSC.

Both of them are consumed here:

https://github.com/xamarin/xamarin-macios/pull/3246/commits/44fbb24bd04071fd5574316577033419961bb49d#diff-f57f2991a6aa25fe45d8036c51bf8b4dR499

and the next diff with $$(MAC_$(1)_CSC) being full or mobile, depending on template instancing.